src/main/java/frc5124/robot2020/commands/turret/RotateTurret.java

```diff
@@ -40,12 +41,16 @@ public void initialize() {
   @Override
   public void execute() {
     if (operatorRight.get()) {
-      turret.directPower(.2);
-    }else if (operatorLeft.get()) {
       turret.directPower(-.2);
+    }else if (operatorLeft.get()) {
+      turret.directPower(.2);
     } else {
       turret.directPower(0);
     }
+
+    
+
+    SmartDashboard.putNumber("TurretDegrees", turret.getDegrees());
   }
 
   // Called once the command ends or is interrupted.
```

Comment:
This shouldn't be a single command that accepts buttons and uses them. It should be a pair of while held commands, one for each direction.